pom.xml

```diff
@@ -239,6 +251,30 @@
     </pluginRepositories>
     <build>
         <plugins>
+            <!-- JaCoCo plugin -->
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.10</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                </executions>
```

Comment:
We can use this plugin in the pom.xml of runtime to execute a report-aggregate task which can aggregate the test coverage of all modules.

Reply:
ok,I have modified